### ISSUE TITLE
Error: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'status' cannot be null

### ISSUE THREAD

**Issue by boehmgui** [freescout-helpdesk/freescout#3050]
For some time (I don't know when it started) I find several following messages in the logs:

```
Error: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'status' cannot be null (SQL: insert into `conversations` (`preview`, `type`, `state`, `subject`, `mailbox_id`, `customer_id`, `created_by_customer_id`, `source_via`, `source_type`, `has_attachments`, `cc`, `customer_email`, `status`, `last_reply_at`, `last_reply_from`, `folder_id`, `number`, `updated_at`, `created_at`) values (Hallo!Das ist Martin von USA DIGITIZING LLC. WARUM USA D Ha DIGITIZING LLC IN BETRACHT ZIEHEN1 Wir versichern Ihnen, dass Sie die gute Qualität und auch die bestmögliche Digitalisierung nach Ihren Bedürfnissen erhalten. Dann sind Sie bei uns genau richtig, 2, Holen Sie sich ein beliebiges linkes Brust- / Kappenlogo für 5 Euro, 1, 7892, 7892, 1, 1, 1, , martin@usadigit.com, , 2023-05-10 21:35:01, 1, 1, 621, 2023-05-10 21:35:01, 2023-05-10 21:35:01, ?)); File: /var/www/vhosts/xxxx/xxxxx/vendor/laravel/framework/src/Illuminate/Database/Connection.php (664)
```

I can't find any of these emails from the logs in the inbox. 
These emails all have in common that they are actually spam. 

PHP version: 8.1.19
FreeScout version: 1.8.76
Spam Module: 1.0.15

**Comment by freescout-helpdesk** [comment]
Please share the full error message.

**Comment by boehmgui** [comment]
Copied that from the web interface, could you please point me to its full version?
Apologies for looking stupid....

**Comment by freescout-helpdesk** [comment]
Check in Manage > Logs > App Logs.

**Comment by boehmgui** [comment]
you mean fetch-emails.log?

```
SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'status' cannot be null (SQL: insert into `conversations` (`preview`, `type`, `state`, `subject`, `mailbox_id`, `customer_id`, `created_by_customer_id`, `source_via`, `source_type`, `cc`, `customer_email`, `status`, `last_reply_at`, `last_reply_from`, `folder_id`, `number`, `updated_at`, `created_at`) values (Werkzeug 69,- Wenn dieser Newsletter nicht richtig angezeigt wird, klicken Sie bitte hier.Komplettset: 69,- Kompletter Werkzeugkoffer. 69,- Euro zzgl. 19% MwSt. Zum Shop Jetzt hat man immer alles was man braucht ordentlich sortiert!Geeignet für Haus, Gart, 1, 2, Nur 69,- Euro netto:188-teiliges Komplett-Werkzeugset mit Allem von A-Z., 1, 4, 4, 1, 1, , info@premiumbox-handel.de, , 2023-05-26 13:32:01, 1, 1, 694, 2023-05-26 13:32:01, 2023-05-26 13:32:01)); File: /var/www/vhosts/xxxx/xxxxx/vendor/laravel/framework/src/Illuminate/Database/Connection.php (664)
```
doesn't look much different to me....

**Comment by freescout-helpdesk** [comment]
In `laravel-xxxx-xx-xx.log`

**Comment by boehmgui** [comment]
I don't have that - only:
![image](https://github.com/freescout-helpdesk/freescout/assets/102216227/7d7131ca-3918-4d77-8b6b-7e6419b2a8b8)

**Comment by freescout-helpdesk** [comment]
Does it happen with disabled "Spam Filter Module"?

**Comment by freescout-helpdesk** [comment]
Fixed in Spam Filter Module v1.0.16
